e2e/wdio/core/tests/illustrated-message.e2e-spec.ts

```diff
@@ -0,0 +1,96 @@
+import { IllustratedMessagePo } from '../pages/illustrated-message.po';
+import {checkElementScreenshot, 
+    click, 
+    mouseHoverElement, 
+    saveElementScreenshot, 
+    isElementClickable, 
+    elementDisplayed,
+    pause,
+    doesItExist} from '../../driver/wdio';
+
+describe('Illistrated-message tests', function() {
+    const illustratedMessagePage = new IllustratedMessagePo();
+    const {sceneFirstBtn, 
+        sceneSecondBtn, 
+        spotBtn, 
+        buttonDialog, 
+        dialogPopup,
+        closePopupButtonHeader,
+        closePopupButton} = illustratedMessagePage;
+
+    beforeAll(()=>{
+        illustratedMessagePage.open();                              
+    },1);
+
+    it('should check RTL and LTR orientation', () => {              
+        illustratedMessagePage.checkRtlSwitch();
+    });
+
+    it('should check visual regression for all examples', () => {
+        illustratedMessagePage.saveExampleBaselineScreenshot();
+        expect(illustratedMessagePage.compareWithBaseline()).toBeLessThan(1);
+    });
+
+    it('should check is button clickable', () => {
+        expect(isElementClickable(sceneFirstBtn)).toBeTruthy();
+        expect(isElementClickable(sceneSecondBtn)).toBeTruthy();
+        expect(isElementClickable(spotBtn)).toBeTruthy();
+    });
+
+    it('should open dialog popup illustrated Message', () => {
+        click(buttonDialog);
+        pause(100);
+        expect(elementDisplayed(dialogPopup)).toBeTruthy();
+    });
+
+    it('should close dialog popup illustrated Message "closePopupButtonHeader"', () => {
+        click(closePopupButtonHeader);
+        expect(doesItExist(dialogPopup)).toBeFalsy();
+    });
+
+    it('should close dialog popup illustrated Message "closePopupButton"', () => {
```

Comment:
this is appears in the report logs, need to be more readable for all not only code owners like 
should close dialog popup illustrated Message by click on "Close" button. etc.